ui/dev.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:16.10-alpine3.11
+WORKDIR /app
+
+COPY package.json .
+COPY yarn.lock .
+RUN yarn
+VOLUME [ "/app/node_modules" ]
+
+RUN echo "#!/bin/sh" >> /entry_point.sh
+RUN echo "yarn dev --host" >> /entry_point.sh
+RUN chmod +x /entry_point.sh
+
+CMD ["/entry_point.sh"]
```

Comment:
The entry_point.sh only contains one line, what about just `CMD [ "yarn", "dev", "--host"]`?

Reply:
make sense!